docs/Catalog/Qurator.md

```diff
@@ -0,0 +1,82 @@
+# Qurator Omni Beta
+
+`Qurator Omni` is an always-available AI assistant embedded directly into the
+Quilt web catalog. It allows users to interact with Quilt packages and search
```

Comment:
there's no "interaction with quilt packages" currently implemented (aside from searching them and accessing file/entry contents)

Reply:
"S3 buckets"